list-agreements.lua

```diff
@@ -1,120 +1,218 @@
--- Lists all guildhall and temple agreements
+-- Lists outstanding Guildhall and Temple agreements with Org. Names
 
 --[====[
 
 list-agreements
```

Comment:
Check out a few other scripts to see how these help blurbs are usually formatted. Here are some relevant examples: https://github.com/DFHack/scripts/blob/master/build-now.lua and https://github.com/DFHack/scripts/blob/master/prioritize.lua